AnkiDroid/src/main/java/com/ichi2/anki/CardTemplateBrowserAppearanceEditor.kt

```diff
@@ -98,7 +98,9 @@ class CardTemplateBrowserAppearanceEditor : AnkiActivity() {
     }
 
     private fun showDiscardChangesDialog() {
-        DiscardChangesDialog.showDialog(this, ::discardChangesAndClose)
+        DiscardChangesDialog.showDialog(this, getString(R.string.discard), CollectionManager.TR.addingKeepEditing(), CollectionManager.TR.addingDiscardCurrentInput()) {
```

Comment:
Now you've added the defaults, you can remove the parameters which use default values

```suggestion
        DiscardChangesDialog.showDialog(this) {
```

Reply:
Error is found : No value passed for parameter 'message'

Reply:
Error is found : No value passed for parameter 'message'

Reply:
Error found : No value passed for parameter 'message'